samples_project/ProjectSettings/GraphicsSettings.asset

```diff
@@ -39,7 +39,7 @@ GraphicsSettings:
   m_PreloadedShaders: []
   m_PreloadShadersBatchTimeLimit: -1
   m_SpritesDefaultMaterial: {fileID: 10754, guid: 0000000000000000f000000000000000, type: 0}
-  m_CustomRenderPipeline: {fileID: 11400000, guid: 7d6fa941b3ce8564ba81a91532bdfe77, type: 2}
+  m_CustomRenderPipeline: {fileID: 11400000, guid: 95808ba49a714934781374cde9f599fd, type: 2}
```

Comment:
The pipeline has to be changed project-wide as far as I know...not sure of a way to do it for a specific scene.  Rather than changing the pipeline, though, the viewshed-specific renderers could just be added to the existing sample pipelines and only used by the cameras in that scene